INSTALL

```diff
@@ -1,5 +1,31 @@
-Installing Iris currently requires you to build it from source. The 
-latest release is available from https://github.com/SciTools/iris.
+You can either install Iris using the conda package manager or from source.
+
+Installing from conda
+---------------------
+
+Iris can be installed using the `conda <http://conda.pydata.org/>`_ package manager.
```

Comment:
I see what you mean now.  the link conversion does work

i'll remove the link markup, it's not required